src/mtype.d

```diff
@@ -4747,7 +4747,7 @@ extern (C++) final class TypeSArray : TypeArray
             if (d1 != d2)
             {
             Loverflow:
-                error(loc, "%s size %llu * %llu exceeds 16MiB size limit for static array", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
+                error(loc, "Static array size too big", toChars(), cast(ulong)tbn.size(loc), cast(ulong)d1);
```

Comment:
This looks like a degradation of the previous error message. Not even the name of the array is mentioned any more. Also, all extra parameters passed are unused. It'd be nice to show what the value of the limit "too big".